Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -153,23 +156,32 @@ public void ClimbSurface()
 	{
 		if(climbSurface)
 		{
-			rigidbody.useGravity = false;
-			currStoppingPower = stoppingSpeed;
+			ClimbableTag ct = climbSurface.GetComponent<ClimbableTag>();
+			if(ct)
+			{
+				rigidbody.useGravity = false;
+				currStoppingPower = stoppingSpeed;
 
-			Vector3 adjInput = climbSurface.InverseTransformDirection( inputVec );
-			moveVec = transform.rotation * adjInput * climbMoveSpeed * moveSpeedMod;
+				Vector3 adjInput = inputVec;
```

Comment:
I read it as a adjusted input, but I'll make it something that makes more sense.